src/validation.h

```diff
@@ -529,6 +529,15 @@ class CoinsViews {
     void InitCache() EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
 };
 
+enum class CoinsCacheSizeState
+{
+    //! The coins cache is in immediate need of a flush.
+    CRITICAL = 2,
```

Comment:
Why you choose this order (2,1,0)?

Reply:
Why not? It's so we can make statements like https://github.com/bitcoin/bitcoin/pull/15606/commits/f8a21cd1dbde9389523e21a93c0f0b1734336996#diff-24efdb00bfbe56b140fb006b562cc70bR5331

Reply:
Ok.